lab_1/main.py

```diff
@@ -9,10 +9,19 @@ def tokenize(text: str) -> list:
     Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
     :param text: the initial text
     :return: a list of lowercased tokens without punctuation
-    e.g. text = 'The weather is sunny, the man is happy.'
+    E.G.. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    new_text = ""
+    text = text.lower()
+    trash = set("""1234567890-=!@#$%^&*()_+,./<>?;:'"[{}]"'""")
```

Comment:
it is not trash. These are characters to remove from the string

Reply:
should it be renamed or should I remake the whole function?

Reply:
no, just rename